IFC4x3/Quantities/f/FootPrintArea_2$0cTYdvf6QO8z1nvVIBpB/DocQuantity.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FootPrintArea_20cTYdvf6QO8z1nvVIBpB" Name="FootPrintArea" UniqueId="bf026762-9f9a-4669-823d-071e5f48bccb" />
```

Comment:
1. Why not extract this from the model?
2. Is this a quantity?
3. No datatype set.

Reply:
Right, it's a quantity! For the datatype, there is an issue on ifcdoc that didn't update the file... 
So I did it manualy. Commit : 48eb2f882610fb71dfd3c751fdc9e03736d59a7b